tests/lib/bin/TODO

```diff
@@ -0,0 +1,76 @@
+ONCE THINGS ARE RENAMED/MADE UNIFORM THIS SHOULD BECOME A README WITHOUT THE TODO PART!
+
+This directory collects useful utility commands that can be used in
+our spread test suites. (They are separate from the shell function
+libraries *.sh in the level above.)
+
+Commands here should have names that differentiate them from most
```

Comment:
Why should they differentiate?

Reply:
You created most of the *-tool, so you know what is which. If I look at a test I want a quick sense of what might come from where and to go look what it does given that there's a limit how self-describing these things can be,  shell functions use "_",  the tools are very unclear. Making them look like something that could live in */bin dirs one day is an anti-goal for me. Consider also that we are very multi distro so it's hard to guess what could be or not be in */bin.

Reply:
I also want to try to pre-avoid clashes if possible.